bftengine/src/preprocessor/tests/preprocessor_test.cpp

```diff
@@ -0,0 +1,227 @@
+// Copyright 2019 VMware, all rights reserved
+/**
+ * Test public functions for RocksDBMetadataStorage class.
+ */
```

Comment:
nit - the copyright message seems incomplete

Reply:
You are right, copy-paste from the wrong place.